src/states/elder/mod.rs

```diff
@@ -1812,6 +1821,11 @@ impl Elder {
     ) {
         self.send_message_to_targets(dst_targets, dg_size, message)
     }
+
+    pub fn vote_for_prune(&mut self) {
```

Comment:
Use `vote_for_event` instead of this new funcion?

Reply:
using `vote_for_event` means need to expose extra NetworkEvent and LogIndent, which I think don't worth it?

Reply:
No make sense, can use it from within vote_for_prune though.